client/src/components/assessments/AssessmentResultsTable.js

```diff
@@ -40,7 +41,10 @@ const EntityAssessmentResults = ({
   if (!entity) {
     return null
   }
-  const instantAssessmentConfig = entity.getInstantAssessmentConfig()
+  const instantAssessmentConfig = Model.filterAssessmentConfig(
+    entity.getInstantAssessmentConfig(),
+    entity // TODO: add relatedObject?
+  )
```

Comment:
Left so we don't forget; I'm not 100% sure it is useful here, esp. regarding the aggregations.

Reply:
I have added extra condition for `relatedObject`, so that when we are aggregating the assessment questions, we don't filter `relatedObject` specific questions.